ihp-ide/IHP/IDE/SchemaDesigner/View/Layout.hs

```diff
@@ -404,7 +404,16 @@ renderColumn Column { name, columnType, defaultValue, notNull, isUnique } id tab
                 Just value -> [hsx|default: {compileExpression value} |]
                 Nothing -> mempty
         renderForeignKey = case findForeignKey statements tableName name of
-            Just addConstraint@AddConstraint { constraint = ForeignKeyConstraint { name = Just constraintName, referenceTable } } -> [hsx|<a href={EditForeignKeyAction tableName name constraintName referenceTable} class="d-block nounderline">FOREIGN KEY: {referenceTable}</a>|]
+            Just addConstraint@AddConstraint { constraint = ForeignKeyConstraint { name = Just constraintName, referenceTable, onDelete = onDeleteConstraint } } -> [hsx|<a href={EditForeignKeyAction tableName name constraintName referenceTable} class="d-block nounderline">FOREIGN KEY: {referenceTable} (On Delete: {onDeleteText})</a>|]
+                where
+                onDeleteText :: Text
+                onDeleteText = case onDeleteConstraint of
```

Comment:
why not use tshow or https://github.com/digitallyinduced/ihp/blob/master/ihp-ide/IHP/IDE/SchemaDesigner/Compiler.hs#L93 here? right now it's adding a bit of duplication